clients/vault/tests/helper/constants.rs

```diff
@@ -25,7 +33,7 @@ pub const LESS_THAN_4_CURRENCY_CODE: CurrencyId = CurrencyId::AlphaNum4(
 
 #[allow(dead_code)]
 pub const DEFAULT_MAINNET_DEST_SECRET_KEY: &'static str =
-	"SCJ7XV73Q642EPMUMSPO5ECOXWTMWR52MGPMWT6ELV3VUFPH653IOEUS";
+	"SBLI7RKEJAEFGLZUBSCOFJHQBPFYIIPLBCKN7WVCWT4NEG2UJEW33N73";
```

Comment:
It is, and it has to because it needs to have trustlines to some assets established in order to receive them 😬 
We might want to give [this](https://github.com/pendulum-chain/spacewalk/issues/471) task a higher priority to clean it all up but it's not that easy since we use a lot of different secrets which would need to be streamlined first.

Reply:
Oh okay, well we may want to delete this comment for now 😂.